app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java

```diff
@@ -50,8 +49,64 @@ public Meeting(int id, Date start_date, Date end_Date, JSONObject goals,
         this.supervisor = supervisor;
         this.created_at = created_at;
         this.updated_at = updated_at;
+    }
 
-
+    /**
+     * This method is called when a user requests to see a meeting details.
+     * @param {int}          id            [Meeting id].
+     * @param {HTTPResponse} httpResponse  [Interface instance].
+     * @param {String}       token         [Authorization token].
+     */
+    public static void getMeeting(int id, final HTTPResponse httpResponse, String token){
+
+        String stringId = String.valueOf(id);
+        OkHttpClient client = new OkHttpClient();
+        Request request = new Request.Builder()
+                .header("Authorization",token)
+                .header("user_agent","Android")
+                .url("http://ieeeguc.org/api/meeting/" + stringId)
+                .get()
+                .build();
+        client.newCall(request).enqueue(new Callback() {
+            @Override
+            public void onFailure(Call call, IOException e) {
+
+                MainActivity.UIHandler.post(new Runnable() {
+                    @Override
+                    public void run() {
+
+                        httpResponse.onFailure(-1,null);
+                    }
+                });
+                call.cancel();
+            }
+
+            @Override
+            public void onResponse(Call call, Response response) throws IOException {
+
+                try {
+                    String body = response.body().string();
+                    final JSONObject responseBody =  new JSONObject(body);
+                    final int statusCode = response.code();
+                    MainActivity.UIHandler.post(new Runnable() {
+                        @Override
+                        public void run() {
+                            if(statusCode/100 == 2)
+                                httpResponse.onSuccess(statusCode,responseBody);
```

Comment:
you need the else clause as this way both of the methods will be called after each other.